One unsuccessful column instead of three

adwait-godbole opened the pull request:
Fixes #1003 

@Julian let me know of any improvements on this. Thanks!

<!-- readthedocs-preview bowtie-json-schema start -->
----
📚 Documentation preview 📚: https://bowtie-json-schema--1226.org.readthedocs.build/en/1226/

<!-- readthedocs-preview bowtie-json-schema end -->

Julian commented on frontend/src/components/Summary/ImplementationRow.tsx:
```diff
@@ -65,20 +65,20 @@ const ImplementationRow = ({
                   <Col>
-                    <span>
-                      <b style={{ fontSize: "1rem" }}>failed</b>:&nbsp;
+                    <b style={{ fontSize: "1rem" }}>failed:</b>&nbsp;
```
The manual `b` styling here was always a bit off to me, I think while we're here maybe can we change this to be labels? Maybe with a table. Or else maybe a `<hl>`? Or some other idea? I guess `hl` is most obvious to me.

adwait-godbole replied:
`<h1>` is too big for this purpose, going with `<h6>` instead.

Julian commented on frontend/src/components/Summary/ImplementationRow.tsx:
```diff
@@ -51,26 +57,45 @@ const ImplementationRow = ({
+                <Row className="d-flex flex-column gap-2">
+                  <Col>
+                    <b style={{ fontSize: "1rem" }}>failed:</b>&nbsp;
+                    <span style={{ fontWeight: "500" }}>
```
Any reason you're not using the Bootstrap classes for font sizing and weights? https://getbootstrap.com/docs/5.3/utilities/text/#font-weight-and-italics

adwait-godbole replied:
I wanted something that was just slightly less bolder than `fw-bold` and nothing in those options on the docs were satisfying the requirements so I made it 500.

adwait-godbole replied:
It would've been easier if there was a `fw-semi-bold` instead. So had to do 500.

adwait-godbole commented:
@Julian I've pushed some changes and addressed the above comments. Can you take a look at the UI locally once and let me know if any other things that need to be changed. Thanks!

adwait-godbole commented:
@Julian I've made some changes and addressed the above comments. Can run please run the UI locally once and confirm if any other things need to be changed ? Thanks!

adwait-godbole commented:
I've made some changes and addressed the above comments. Can run please run the UI locally once and confirm if any other things need to be changed ? Thanks!

adwait-godbole commented:
@Julian I am signing off for today but let me know if there's any other problem here.

Julian commented:
This looks like an improvement I think, thanks!
